.pylintrc

```diff
@@ -1,4 +1,4 @@
 [MASTER]
-disable=R0903,R0901
+disable=R0903,R0901,R0914
```

Comment:
This value does not make any sense at all. Why limit the number of locals? Why not limit the number of globals then? Because in a function's body, we have to think about the global symbols too.

The number of variables is a very bad metric for complexity. There are better tools out there than pylint, like SonarCloud which will look for the amount of nested blocks, the number of arguments, the function length, etc... which are far better indicators of complexity.

We should get rid of pylint, replace it with isort and black (which are formatters that an reformat the code automatically, pylint can't do that), and maybe setup Sonar if you want some more static analysis. But that's for another PR :)